app/Rules/ValidTestQuestionsScore.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class ValidTestQuestionsScore implements Rule
```

Comment:
I think a name **ValidTestQuestionsTotalScore** will describe more accurate the meaning of the class.